tests/integration/balena.spec.ts

```diff
@@ -268,9 +275,10 @@ describe('Balena SDK', function () {
 									'Interceptor response hook should be called',
 								),
 							);
-						}));
+						});
+					});
 
-					describe('to a differnet server', () =>
+					describe('to a differnet server', () => {
```

Comment:
nit: can we fix this typo while we're here? 😄 
```suggestion
					describe('to a different server', () => {
```